src/onDraw/customizeCells.js

```diff
@@ -1,7 +1,7 @@
 export default function customizeCells() {
     // add Dynel's hover text to table headers
     d3
-        .select('th.answer_query_cnt')
+        .select('th.answer_query_ct')
```

Comment:
I don't love the idea of hard-coding a variable name instead of referencing a setting here.  However, the settings don't really distinguish form-level variables from query-frequency variables so we should revisit the settings in the future.

Reply:
Yeah I don't either, it's kind of weird to go outside of the settings to change a variable name